### PR TITLE
Add ncdu package.

### PR DESCRIPTION
**Pull request by netflash**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This file is used to list changes made in each version of the devopsbox cookbook.
 
+## Master
+ * install ncdu package.
+
```
Comment (toddmichael) [inline CHANGELOG.md]: no version bump?

File: recipes/default.rb [hunk 2]
```diff
@@ -29,6 +29,7 @@
   wget
   zip
   unzip
+  ncdu
```
Comment (toddmichael) [inline recipes/default.rb]: list alphabetically to keep consistent?

File: CHANGELOG.md (hunk 1 above)
Comment (strata-alex) [inline CHANGELOG.md]: Not yet, as we don't publish this cookbook to supermarket. BTW are we going to ?

File: recipes/default.rb (hunk 2 above)
Comment (strata-alex) [inline recipes/default.rb]: done

File: CHANGELOG.md (hunk 1 above)
Comment (toddmichael) [inline CHANGELOG.md]: Haven't really thought about that, to be honest.  IMHO, we should bump versions and use Git tags on Master whenever we make a changes, regardless, as failing to do so could break things for cookbook users who are pulling directly via GitHub/Berksfile.

Comment (netflash) [inline CHANGELOG.md]: Agree. But version bump should be done directly in master, after merging any PR

**Pull request merged**